TradeItIosTicketSDK2/TradeItPortfolioBalanceEquityPresenter.swift

```diff
@@ -44,7 +44,7 @@ class TradeItPortfolioBalanceEquityPresenter {
         var dayReturnString = ""
 
         if let dayAbsoluteReturn = self.balance?.dayAbsoluteReturn {
-            dayReturnString = NumberFormatter.formatCurrency(dayAbsoluteReturn, displayVariance: true, currencyCode: balance?.accountBaseCurrency)
+            dayReturnString = NumberFormatter.formatCurrency(dayAbsoluteReturn, displayVariance: true)
```

Comment:
Why remove this? If the balance returned is not USD then this won't work.

Reply:
ah shizen, good catch. had mean to circle back with @falnatsheh and think of the best way to handle this.

Options that first come to mind are:
1. `dayReturnString = NumberFormatter.formatCurrency(dayAbsoluteReturn, displayVariance: true, currencyCode: balance?.accountBaseCurrency ?? TradeItPresenter.DEFAULT_CURRENCY_CODE` 
1. Swap out the following code with `currencyFormatter.currencyCode = currencyCode ?? TradeItPresenter.DEFAULT_CURRENCY_CODE`
https://github.com/tradingticket/TradeItIosTicketSDK2/blob/be032b9ebaf53871795758433ba8a032d7cafe28/TradeItIosTicketSDK2/NumberFormatter.swift#L13

I think I'd lean toward option 2.

Reply:
Maybe the right thing to do is to ignore the value if we can't determine a currency code? Just return `nil`.

Reply:
good point -- take a look at https://github.com/tradingticket/TradeItIosTicketSDK2/pull/351/commits/a2434727fbda466a1386e9362185a8fcadbac142 and lmk if I'm Swift-ing right